### ISSUE TITLE
README needs to be clearer on manual procedure vs automated ansible procedure

### ISSUE THREAD

**Issue by itewk** [openshift/request-header-saml-service-provider#27]
@briantward with all of your latest updates that @brenton just merged, reading through the readme I can no longer tell the correct procedure to follow to get this up an running in production.

I think it would be best to seperate the read me into three distinct sections:

1) shared setup
2) automated ansible install (new steps)
3) manual install (previous steps)

And then I think maybe moving the RH-SSO parts to a completely different doc since no one will or should use those in production.

I think the key is our main read me story should be something that is useable in production but now if I am reading the doc I can't tell which steps I need to actually do to get things running as the ansible, rh-sso, and the other manual steps are all mixed together.

**Comment by briantward** [comment]
That's a fair point.  I'm open to conversation and reorganization, as I wasn't convinced it was the best approach either.  I originally separated the steps, but then as I was trying to update the manual steps I found it useful to have the ansible script nearby for reference, which is why I added that to the bottom of each section.   I'm not sure I understand what you mean by "shared setup".  Anyways, I do think you're right in that it can be broken down better into separate parts.  You want to schedule some time to work on it together?
